ResearchKit/Common/ORKFormStepViewController.m

```diff
@@ -890,9 +915,7 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
         ORKTableSection *section = _sections[indexPath.section];
         ORKTableCellItem *cellItem = section.items[indexPath.row - 1];
         [section.textChoiceCellGroup didSelectCellAtIndexPath:indexPath];
-        
-        id answer = (cellItem.formItem.questionType == ORKQuestionTypeBoolean)? [section.textChoiceCellGroup answerForBoolean] : [section.textChoiceCellGroup answer];
-        
+        id answer = ([cellItem.formItem.answerFormat isKindOfClass:[ORKBooleanAnswerFormat class]])? [section.textChoiceCellGroup answerForBoolean] : [section.textChoiceCellGroup answer];
```

Comment:
I see, so is this a bug currently? How are boolean answers recorded in a form in the current RK?